Public/Src/Cache/ContentStore/Library/Sessions/ContentSessionBase.cs

```diff
@@ -32,14 +32,17 @@ namespace BuildXL.Cache.ContentStore.Sessions
     /// </summary>
     public abstract class ContentSessionBase : StartupShutdownBase, IContentSession
     {
-        private CounterCollection<ContentSessionBaseCounters> _counters { get; } = new CounterCollection<ContentSessionBaseCounters>();
+        private readonly CounterCollection<ContentSessionBaseCounters> _counters;
 
         /// <inheritdoc />
         public string Name { get; }
 
         /// <nodoc />
         protected virtual bool TracePinFinished => true;
 
+        /// <nodoc />
+        protected virtual bool TraceErrorsOnly => false;

```

Comment:
Will we not do a virtual property for TraceOperationStart, so that DistributedContentStore still logs starts?

Reply:
If TraceErrorsOnly is true, then the base session won't trace anything at all: no starts or stops.

Reply:
Yeah, I meant an entirely different property; I only put the comment here because it seemed the logical place where the new property would live.

Reply:
Got it. Added another property.